generators/generator-base.js

```diff
@@ -808,7 +808,28 @@ module.exports = class extends PrivateBase {
      * Generate a date to be used by Liquibase changelogs.
      */
     dateFormatForLiquibase() {
-        const now = new Date();
+        let now = new Date();
+        // Run reproducible timestamp when regenerating the project with with-entities option.
+        if (this.options.withEntities) {
+            if (this.configOptions.lastLiquibaseTimestamp) {
+                // Counter already started.
+                now = this.configOptions.lastLiquibaseTimestamp;
+            } else {
+                // Create a new counter
+                let creationTimestamp;
+                if (this.options.creationTimestamp) {
+                    creationTimestamp = Date.parse(this.options.creationTimestamp);
```

Comment:
I wonder if [this](https://stackoverflow.com/questions/2587345/why-does-date-parse-give-incorrect-results) applies, what do you think?

Reply:
Seems that the following fixes this problem.
`const nowUTC = new Date(...`

Reply:
yup